statsmodels/iolib/summary2.py

```diff
@@ -556,10 +569,8 @@ def merg(x, y):
     index = summ.index.get_level_values(0)
     for i in range(0, index.shape[0], 2):
         idx.append(index[i])
-        if (i + 1) < index.shape[0] and (index[i] == index[i + 1]):
-            idx.append("")
-        else:
-            idx.append(index[i + 1])
+        if (i + 1) < index.shape[0]:
+            idx.append(index[i + 1] if index[i] != index[i + 1] else "")
```

Comment:
AFAICS This changes the else path when `(i + 1) == index.shape[0]`
which version is correct?

Reply:
answer in commit "fix: Allow odd number of elements in index"

Reply:
Indeed (db364565 for ref).

Currently, if `(i+1) < index.shape[0] is False`, the `else` clause will still try to access `index[i+1]`.